src/sql/engine/dml/ob_link_op.h

```diff
@@ -78,7 +78,7 @@ class ObLinkOp : public ObOperator
   int64_t next_sql_req_level_;
   static const int64_t STMT_BUF_BLOCK;
   common::sqlclient::DblinkDriverProto link_type_;
-  bool in_xa_trascaction_; // is dblink write/read remote database in xa trasaction
+  bool in_xa_trascaction_; // is dblink write/read remote database in xa transaction
```

Comment:
`in_xa_transaction`?

Reply:
Hey @dentiny, just noticed that **in_xa_trascaction** appears in 3 different files, Would it affect anything else if I fix the variable name?